docs/hello_nf-core/05_input_validation.md

```diff
@@ -107,9 +106,54 @@ graph LR
 
 Validation happens **before** any pipeline processes run, providing fast feedback and preventing wasted compute time.
 
+### 1.4. Configure validation to skip input file validation
+
+The nf-core pipeline template comes with nf-schema already installed and configured:
+
+- The nf-schema plugin is installed via the `plugins{}` block in `nextflow.config`
+- Parameter validation is enabled by default via `params.validate_params = true`
+- The validation is performed by the `UTILS_NFSCHEMA_PLUGIN` subworkflow during pipeline initialization
+
+The validation behavior is controlled through the `validation{}` scope in `nextflow.config`.
+
+Since we'll be working on parameter validation first (section 2) and won't configure the input data schema until section 3, we need to temporarily tell nf-schema to skip validating the `input` parameter's file contents.
+
+Open `nextflow.config` and find the `validation` block (around line 246). Add `ignoreParams` to skip input file validation:
+
+=== "After"
+
+    ```groovy title="nextflow.config" hl_lines="3" linenums="246"
+    validation {
+        defaultIgnoreParams = ["genomes"]
+        ignoreParams = ['input']
+        monochromeLogs = params.monochrome_logs
+    }
+    ```
+
+=== "Before"
+
+    ```groovy title="nextflow.config" linenums="246"
+    validation {
+        defaultIgnoreParams = ["genomes"]
+        monochromeLogs = params.monochrome_logs
+    }
+    ```
+
+This configuration tells nf-schema to:
+
+- **`defaultIgnoreParams`**: Skip validation of complex parameters like `genomes` (set by template developers)
+- **`ignoreParams`**: Skip validation of the `input` parameter's file contents (temporary - we'll remove this in section 3)
+- **`monochromeLogs`**: Control colored output in validation messages
+
+!!! note "Why ignore the input parameter?"
+
+    The `input` parameter in `nextflow_schema.json` has `"schema": "assets/schema_input.json"` which tells nf-schema to validate the *contents* of the input CSV file against that schema.
+    Since we haven't configured that schema yet, we temporarily ignore this validation.
+    We'll remove this setting in section 3 after configuring the input data schema.
+
 ### Takeaway
 
-You now understand what nf-schema does, the two types of validation it provides, and when validation occurs in the pipeline execution lifecycle.
+You now understand what nf-schema does, the two types of validation it provides, when validation occurs, and how to configure validation behavior. You've also temporarily disabled input file validation so we can focus on parameter validation first.
```

Comment:
I feel like "You now understand what nf-schema does" is a bit of an overstatement. You've ntroduced me to it, but I think I'll wait until I've tried it in practice to claim I've actually understood it 😅